baram/efs_manager.py

```diff
@@ -0,0 +1,78 @@
+import boto3
+
+from baram.log_manager import LogManager
+
+
+class EFSManager(object):
+    def __init__(self):
+        self.cli = boto3.client('efs')
+        self.logger = LogManager.get_logger()
+
+    def list_file_systems(self):
+        """
+        List one or more of your EFS.
+
+        :return:
+        """
+        return self.cli.describe_file_systems()['FileSystems']
+
+    def list_mount_targets(self, file_system_id: str):
+        """
+        List mount targets of specific file system.
+
+        :param file_system_id: FileSystemId
+        :return:
+        """
+        try:
+            return self.cli.describe_mount_targets(FileSystemId=file_system_id)['MountTargets']
+        except:
+            self.logger.info('error')
+
+    def delete_redundant_file_systems(self, redundant_domain_ids: list = []):
+        """
+        Delete file systems of disused domains.
+
+        :param redundant_domain_ids: list of DomainId of redundant domains
+        :return:
+        """
+        redundant_fs_ids = [fs['FileSystemId'] for fs in self.list_file_systems()
+                            if fs['CreationToken'] not in redundant_domain_ids
```

Comment:
Can I ask what this condition is checking?

Reply:
You'd probably notice why this condition exists via `aws ec2 describe-file-systems`. `CreationToken` is equal to related domain id of each EFS.